client/components/post-schedule/style.scss

```diff
@@ -138,6 +138,10 @@ input[type="text"].post-schedule__clock-time {
 	border-top: 1px solid lighten( $gray, 25% );
 }
 
+.post-schedule__component-timezone {
+	width: 100%;
```

Comment:
This is only used in the example; does it make sense to be in the component styles?